### PR TITLE
fix: updated search info and no result design

### PR DESCRIPTION
**Pull request by muhammadadeeltajamul**
Updated design as per figma mockup

Ticket Link:
https://2u-internal.atlassian.net/browse/INF-412

### CODE REVIEW

File: src/discussions/posts/NoResults.jsx [hunk 1]
```diff
@@ -12,17 +12,21 @@ function NoResults({ intl }) {
...
+  } else if (topicsFilter || learnersFilter) {
+    helpMessage = messages.differentKeywords;
   }
 
   return (
     <div className="h-100 mt-5 align-self-center mx-auto w-50 d-flex flex-column justify-content-center text-center">
-      <h4>{intl.formatMessage(messages.noResultsFound)}</h4>
-      <small>{intl.formatMessage(helpMessage)}</small>
+      <h4 className={topicsFilter || learnersFilter ? 'font-weight-normal text-primary-500' : ''}>{intl.formatMessage(messages.noResultsFound)}</h4>
+      <small className={topicsFilter || learnersFilter ? 'font-weight-normal text-gray-700' : ''}>{intl.formatMessage(helpMessage)}</small>
```
Comment (awais-ansari) [inline src/discussions/posts/NoResults.jsx:29]: use `classnames` for assign classes conditionally

**Review by awais-ansari — APPROVED** [review]

**Review by mehaknasir — APPROVED** [review]
just the change requested by Awais rest looks good to me.

**Review by AhtishamShahid — APPROVED** [review]
Please add a screenshot in the description, Other than that looks good.

**Comment by asadazam93** [comment]
@muhammadadeeltajamul can you please fix the conflicts?

**Pull request merged**